src/components/educationLayout/EducationLayout.tsx

```diff
@@ -11,45 +10,51 @@ function EducationLayout({
   maxCountOfColumns,
   control,
 }: IEducationLayoutProps) {
-  const [columns, setColumns] = useState<{ id: string }[]>([{ id: "initial" }]);
-
-  const { append, remove } = useFieldArray({ control, name: type });
+  const { append, remove, fields } = useFieldArray({ control, name: type });
 
   const handleAddColumn: MouseEventHandler = () => {
-    setColumns((state) => [...state, { id: uuidv4() }]);
+    append({
+      jobTitle: "",
+      employer: "",
+      startDate: null,
+      endDate: null,
+      city: "",
+      description: "",
+      degree: "",
+      school: "",
+      active: true,
+    });
```

Comment:
can we set null insted this full object?

Reply:
fixed